server/lists/adapters/__tests__/workstacks.spec.js

```diff
@@ -17,6 +17,8 @@ const mockFromStaticList = jest.fn((list) => {
     }
 });
 
+const todaysDate = new Date().toISOString().slice(0,10); // Returns todays date in format (yyyy-MM-dd)
```

Comment:
Splitting on `T` might be more reliable.

Reply:
That would return an array, and we don't care about the time. So slicing by the first 10 characters on a string that's a ISO standard is simpler. If you are adamant on this change I can do it.

Reply:
This has been rectified 👍🏼